src/index.js

```diff
@@ -207,28 +207,23 @@ export default function undoable (reducer, rawConfig = {}) {
     clearHistoryType: rawConfig.clearHistoryType || ActionTypes.CLEAR_HISTORY
   }
 
-  return (state, action = {}) => {
+  return (state = config.history, action = {}) => {
     debugStart(action, state)
 
-    let history
+    let history = state
```

Comment:
I think we want history to equal state unconditionally. It may get overwritten inside the `if (!config.history)` statement, but that will only happen a limited number of times when the reducer is un-initialized. Most of the time, we just want to make `history = state`.